student_survey.php

```diff
@@ -179,6 +179,7 @@ function select_questions($choices, $list_of_quesitons)
  */
 function handle_post($body)
 {
+    $body = json_decode(json_encode($body), true);
```

Comment:
Why do you need to encode and then decode?

Reply:
$body is a stdClass. According to https://stackoverflow.com/questions/18576762/php-stdclass-to-array I need to encode then decode $body to convert stdClass to an array.

Reply:
Hunt down where that `stdClass` is coming from. We shouldn't be using that in our code.

Reply:
it comes from line 14. The input stream returns a string which when I decode, gets converted to a stdClass. There's not a good way to easily parse the body.

Reply:
Add a `, true` to the `json_decode`

Reply:
Thank you